### ISSUE TITLE
Term Request: Corneal epithelial microcysts

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11740]
**Preferred term label:**
Corneal epithelial microcysts

**Synonyms**
* Corneal intraepithelial microcysts
* Microcystic changes of corneal epithelium
* Cogan dots

**Definition (free text, please give PubMed ID)**
Presence of tiny, round-to-oval, transparent or putty-gray vesicles or pseudocysts scattered within the corneal epithelium. On retroillumination, they appear as solitary clear cysts or aggregated clusters containing degenerated epithelial cellular debris or intracytoplasmic fibrogranular material (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal epithelium morphology (HP:0011495)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Meesmann corneal dystrophy (MECD) (ORPHA:98954, OMIM:122100, OMIM:618767)
* Lisch epithelial corneal dystrophy (LECD) (ORPHA:98955, OMIM:620763)
* Epithelial basement membrane dystrophy (EBMD) (ORPHA:98956, OMIM:121820)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
